jekyll-dev.yml

```diff
@@ -147,6 +147,7 @@ mesh_product_name: Kong Mesh
 kic_product_name: Kubernetes Ingress Controller
```

Comment:
Can you also change the layout of `'getting-started-guide'` to `'docs-v2'` in both of the jekyll.yml data files? That way it'll pick up all of these layout changes.